app.py

```diff
@@ -68,18 +68,37 @@ def delete_book(connection, book_id):
     return "", 204
 
 
+def update_book(connection, book_id):
+    body = request.json
+
+    new_book_name = body["name"]
+
+    if new_book_name== "":
+        return {"error": "Book name is required"}, 400
+
```

Comment:
what will happen if we pass book_id, which does not exist? can we somehow process this error?